frontend/src/components/GymPage/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+import GymNavBar from "../GymNavBar";
+import GymStatus from "../GymStatus";
+import GymKeyWith from "../GymKeyWith";
+
+export default function GymPage() {
+  return (
+    <>
+      <GymNavBar />
+      <div style={{ verticalAlign: 'middle', justifyContent: 'center' }}>
+      <GymStatus isOpen />
+      <GymKeyWith />
```

Comment:
General guide for naming, 
if you're in the context of "Gym" already, 
you can forgo the prefix. KeyWith, Status, PageNavBar works.

What do you think?